contents/manual/role-based-access.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Role based access
+sidebar: Docs
+showTitle: true
+availability:
+    free: false
+    selfServe: false
+    enterprise: full
+---
+
+PostHog gives you tools for feature access control and team member groupings with role based access inside of PostHog.
+
+## Organizational defaults
+
+By default, team members in your organization have view and edit access to all resource features (feature flags, experiments, session recordings).
+
+These can be changed under organization settings -> role based access tab. You can set organizational defaults to just "view only" and restrict edit access to certain features.
```

Comment:
this seems to suggest we have access control for experiments and recordings as well? 😅

Reply:
we should eventually right 😁 I think the wording is still vague enough where it doesn't necessarily imply that we have access control for experiments/recordings but that all resources are defaulted to edit, using those features as an example of what a "resource" is